<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1405</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WiiVolume</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/Volume Wiimote Icon.iconset/icon_32x32.png</normaloff>:/images/Volume Wiimote Icon.iconset/icon_32x32.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/images/logo2.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/images/logo1.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/images/logo3.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumberOutput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>120</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>7</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelUnits">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>72</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/images/kg_outline.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frameOpenGL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1405</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionFind_Balance_Board"/>
   <addaction name="separator"/>
   <addaction name="actionWeight"/>
   <addaction name="actionVolume"/>
   <addaction name="separator"/>
   <addaction name="actionLeft_Foot"/>
   <addaction name="actionRight_Foot"/>
   <addaction name="separator"/>
   <addaction name="actionRight_View"/>
   <addaction name="actionFront_View"/>
   <addaction name="actionTop_View"/>
   <addaction name="separator"/>
   <addaction name="actionPreset_Centre_of_Interest"/>
   <addaction name="separator"/>
   <addaction name="actionFull_Screen"/>
   <addaction name="actionHide_Toolbar"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFind_Balance_Board">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Find Wiimote Icon.iconset/icon_32x32.png</normaloff>:/images/Find Wiimote Icon.iconset/icon_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Find Balance Board</string>
   </property>
   <property name="toolTip">
    <string>Click this when you are ready to sync the balance board</string>
   </property>
  </action>
  <action name="actionWeight">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Weigh Wiimote Icon.iconset/icon_32x32.png</normaloff>:/images/Weigh Wiimote Icon.iconset/icon_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Weight</string>
   </property>
   <property name="toolTip">
    <string>Select as output the weight in kilograms</string>
   </property>
  </action>
  <action name="actionVolume">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Volume Wiimote Icon.iconset/icon_32x32.png</normaloff>:/images/Volume Wiimote Icon.iconset/icon_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
   <property name="toolTip">
    <string>Select as output the volume in metres cubed</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit the program</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="toolTip">
    <string>Pop up the preferences dialog box to set default values</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Open Icon.png</normaloff>:/images/Open Icon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a GaitSym XML config file</string>
   </property>
  </action>
  <action name="actionLeft_Foot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Left Foot Icon.png</normaloff>:/images/Left Foot Icon.png</iconset>
   </property>
   <property name="text">
    <string>Left Foot</string>
   </property>
   <property name="toolTip">
    <string>Emulates only have the left foot on the board</string>
   </property>
  </action>
  <action name="actionRight_Foot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Right Foot Icon.png</normaloff>:/images/Right Foot Icon.png</iconset>
   </property>
   <property name="text">
    <string>Right Foot</string>
   </property>
   <property name="toolTip">
    <string>Emulates only have the right foot on the board</string>
   </property>
  </action>
  <action name="actionRight_View">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Right View Icon.png</normaloff>:/images/Right View Icon.png</iconset>
   </property>
   <property name="text">
    <string>Right View</string>
   </property>
   <property name="toolTip">
    <string>View the simulation from the right</string>
   </property>
  </action>
  <action name="actionFront_View">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Front View Icon.png</normaloff>:/images/Front View Icon.png</iconset>
   </property>
   <property name="text">
    <string>Front View</string>
   </property>
   <property name="toolTip">
    <string>View the simulation from the front</string>
   </property>
  </action>
  <action name="actionTop_View">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Top View Icon.png</normaloff>:/images/Top View Icon.png</iconset>
   </property>
   <property name="text">
    <string>Top View</string>
   </property>
   <property name="toolTip">
    <string>View the simulation from the top</string>
   </property>
  </action>
  <action name="actionPreset_Centre_of_Interest">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Centre of Interest Icon.png</normaloff>:/images/Centre of Interest Icon.png</iconset>
   </property>
   <property name="text">
    <string>Preset Centre of Interest</string>
   </property>
   <property name="toolTip">
    <string>Set the centre of interest to its preset value</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Fullscreen Icon.png</normaloff>:/images/Fullscreen Icon.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="toolTip">
    <string>Expands the display to take up the full screen (F key toggles)</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionHide_Toolbar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Hide Toolbar Icon.png</normaloff>:/images/Hide Toolbar Icon.png</iconset>
   </property>
   <property name="text">
    <string>Hide Toolbar</string>
   </property>
   <property name="toolTip">
    <string>Hides the tool bar (H key toggles)</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
